code/lib/manager-api/src/modules/stories.ts

```diff
@@ -54,8 +55,7 @@ export interface SubState {
   storiesHash: API_StoriesHash;
   storyId: StoryId;
   viewMode: ViewMode;
-  storiesConfigured: boolean;
-  storiesFailed?: Error;
```

Comment:
Note we never actually looked at this value anywhere, so to simplify things I just dropped it.

Reply:
Renaming the `storiesConfigured` property is a breaking change, are you sure you want to do it?

There are many projects that use it, and here are a couple: [first](https://github.com/AndrianBalanescu/builder/blob/master/packages/storybook/src/Tab.tsx#L64), [second](https://github.com/stackbit/storybook-addon/blob/13fd96d87a0ae2d2c4015cf9d446ef632f2e706c/src/Panel.jsx#L22), [third](https://github.com/jh3y/storybook-addon-studybook/blob/main/src/components/badge.js#L61)

And apparently [some](https://github.com/Andarist/react-spectrum-v3/blob/master/.storybook-v3/theme.js#L18) [addons](https://github.com/adamyonk/storybook-dark-mode/blob/master/src/Tool.tsx#L106) listen to a `storiesConfigured` event 🤔

Reply:
I guess we could put it back, but not use it ourselves. What about the `storiesErrrored` property? We didn't seem to use it but it could be that addons/etc do?

Would we deprecate them?

It seems better to just have a `ready` property and for it to be a bit vaguer as to what it means ("configured" being incorrect as it turns out).

Reply:
I did not see any `storiesErrrored` property anywhere, not sure what you mean?
I'm fine with the change to `ready`, I just wanted to let you know that this will break people's code, and I wanted to know if it was intended

Reply:
Discussed and the conclusion is people should not be reaching inside our state. If they are they'll need to update in a breaking release (although we might not consider such a change breaking).

Reply:
Apparently there is a user facing hook which gives access to the internal state:
https://storybook.js.org/docs/react/addons/addons-api#storybook-hooks